Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.Data/Services/NotificationSearchService.cs

```diff
@@ -68,6 +69,8 @@ public async Task<NotificationSearchResult> SearchNotificationsAsync(Notificatio
                 var allPersistentProvidersTypes = result.Results.Select(x => x.GetType()).Distinct();
                 transientNotificationsQuery = transientNotificationsQuery.Where(x => !allPersistentProvidersTypes.Contains(x.GetType()));
 
+                transientNotificationsQuery = transientNotificationsQuery.Where(x => !x.Kind.EqualsInvariant(x.Type));
```

Comment:
Why we need this check?

Reply:
![image](https://user-images.githubusercontent.com/8361000/58257605-80f7a300-7d71-11e9-8bab-37fa971a861a.png)